### PR TITLE
Task branch

### PR DESCRIPTION
**Pull request by axferox**

### CODE REVIEW

File: utils/pageObjects/basePage/basePage.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    this.baseUrl = 'https://www.ah.nl';
+    this.path = '';
+    this.Header = new Header();
+  }
+
+  getPageUrl() {
+    return `${this.baseUrl}${this.path}`;
+  }
+
+  getCurrentUrl() {
```
Comment (VladSelikh) [inline utils/pageObjects/basePage/basePage.js]: It should be a separate helper function.

File: utils/pageObjects/basePage/basePage.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    return driver.getCurrentUrl();
+  }
+
+  open() {
+    return driver.get(`${this.baseUrl}${this.path}`);
+  }
+
+  getTitle() {
```
Comment (VladSelikh) [inline utils/pageObjects/basePage/basePage.js]: It should be a separate helper function.

File: utils/pageObjects/baseElements/baseElement.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+class BaseElement {
+  constructor(locator) {
+    this.locator = locator;
+  }
+
+  findElement() {
+    return driver.findElement(this.locator);
+  }
+
+  wait() {
```
Comment (VladSelikh) [inline utils/pageObjects/baseElements/baseElement.js]: It should be a separate helper function.

File: utils/webdriverConfigs/chromeDriver.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+async function chromeConfig() {
+  await driver.manage().window().maximize();
+}
+
+async function quit() {
+  return driver.quit();
+}
+
+async function sleep() {
```
Comment (VladSelikh) [inline utils/webdriverConfigs/chromeDriver.js:13]: What is the difference between this function and the` wait()` method of your `BaseElement` class?

File: utils/pageObjects/baseElements/baseElement.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    driver.wait(until.elementIsVisible(this.locator), 10000);
+  }
+
+  textInput(inputData) {
+    return this.findElement().sendKeys(inputData);
```
Comment (VladSelikh) [inline utils/pageObjects/baseElements/baseElement.js]: There is no need to return this.

File: utils/pageObjects/baseElements/baseElement.js [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+    return this.findElement().isSelected();
+  }
+
+  getElementText() {
+    return this.findElement().getText();
+  }
+
+  click() {
+    this.wait();
+    return this.findElement(this.locator).click();
```
Comment (VladSelikh) [inline utils/pageObjects/baseElements/baseElement.js]: There is no need to return this.

File: utils/pageObjects/basePage/basePage.js [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+  getPageUrl() {
+    return `${this.baseUrl}${this.path}`;
+  }
+
+  getCurrentUrl() {
+    return driver.getCurrentUrl();
+  }
+
+  open() {
+    return driver.get(`${this.baseUrl}${this.path}`);
```
Comment (VladSelikh) [inline utils/pageObjects/basePage/basePage.js]: There is no need to return this.

File: utils/webdriverConfigs/chromeDriver.js (hunk 4 above)
Comment (axferox) [reply utils/webdriverConfigs/chromeDriver.js:13]: First: wait rely on an element
Second: sleep is used for service debugging and development.

File: utils/pageObjects/baseElements/baseElement.js (hunk 6 above)
Comment (axferox) [reply utils/pageObjects/baseElements/baseElement.js]: Not sure, test are failing without return here.

**Review by VladSelikh — APPROVED** [review]

**Pull request merged**